src/cryptonote_core/tx_pool.cpp

```diff
@@ -1085,16 +1089,116 @@ namespace cryptonote
     return true;
   }
 
+
+  //---------------------------------------------------------------------------------
+
+  uint32_t get_rta_hdr_supernode_public_key_offset(const rta_header& rta_hdr)
```

Comment:
1. name is wrong - all signatures belongs to supernodes - some of these supernodes are auth sample
2. at least 6 signatures required as auth sample

Reply:
You can see that it's not about signatures.
Signatures live in another container - "std::vector<rta_signature> rta_signs".

He we get offset to 8 supernode pub-keys inside of rta_header.keys.
Does it make sense now?

Reply:
sorry, I meant `keys` of course. all the keys (except POS_KEY_INDEX) belongs to supernodes (pos proxy and wallet proxy are supernodes as well)

Reply:
Then ok. You're rather right. Will fix it. But all this about p1. Not p.2. P.2 - is not related to the matter.

Reply:
Fixed